.goreleaser.yaml

```diff
@@ -0,0 +1,42 @@
+project_name: csproto
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - id: protoc-gen-fastmarshal
+    main: ./cmd/protoc-gen-fastmarshal
+    binary: protoc-gen-fastmarshal
+    flags:
+      - -trimpath
+    ldflags:
+      - '-X main.version={{ .Version }} -X main.commit={{ .ShortCommit }} -X main.date={{ .Date }} -X main.builtBy=goreleaser'
```

Comment:
if you're building using go 1.18, you can make use of the new [debug.ReadBuildInfo](https://pkg.go.dev/runtime/debug@master#ReadBuildInfo) if you want

Reply:
I looked at the docs and the default `ldflags` is what I was passing in, so I removed it completely.